app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -47,6 +50,11 @@ public class TabFragment extends Fragment {
     private RecyclerView recyclerView;
     private TransAdapter mAdapter1;
 
+    /**
+     * Handles Fragments and transcation in the main page based on the position
+     * @param position
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.